treeherder/config/settings.py

```diff
@@ -310,7 +310,7 @@
 CELERY_TASK_CREATE_MISSING_QUEUES = False
 
 # Celery broker setup
-CELERY_BROKER_URL = env('BROKER_URL')
+CELERY_BROKER_URL = env('BROKER_URL', default='http://localhost:15672')
```

Comment:
Reminder @armenzg 
I'm unsure about this value. Let's leave it for now. Once you make all other changes and I try the PR I will let you know.